src/int.rs

```diff
@@ -691,10 +693,12 @@ impl Int {
     ///
     /// # Errors
     ///
+    /// - If `target_width.try_into()` returns an error.
     /// - If the `target_width` is greater than the current width.
-    pub fn into_truncate<W>(self, target_width: W) -> Result<Int>
+    pub fn into_truncate<W, E>(self, target_width: W) -> Result<Int>
```

Comment:
What is the effect of introducing this additional `E` parameter?
Couldn't we simply return `Result<Int, apint::Error>`?

Reply:
Also changes like these should have really been moved into their own PR.

Reply:
It was needed to avoid colliding with blanket impls. I have decided to not use this signature when I redo this PR, because it required me to use a turbofish in some places, completely defeating the point of it.